docusaurus/docs/key-concepts/ui-extensions.md

```diff
@@ -9,57 +9,74 @@ keywords:
   - links
   - extensions
   - app plugins
+  - extension points
+  - components
+  - content
 sidebar_position: 60
 ---
 
-Use UI extensions to contribute new actions and functionality to the core Grafana UI and other app plugins. 
+Use extensions to contribute new actions and functionality to the core Grafana UI and other app plugins. 
 
-## Understand extensions
+## Understand the extensions ecosystem
 
 The UI extensions framework is built around these concepts:
 
-1. `Extension point` - A place in the UI where plugins can contribute new functionality to the end user. Both the Grafana UI and app plugins can expose extension points. 
-2. `Extension` - New functionality you register to an extension point. For example, an extension can provide a navigational link to bring users to a particular view, open a modal menu allowing the user to configure an action to take from within their current context (for instance, to create a SLO), or trigger background tasks. There are three types of extensions: links, components, and functions.
-3. `Exposed component` - A component from an app plugin you can expose to easily share functionality with other app plugins without having to register to an extension point.  
+- **Extension point**: A place in Grafana Core or in a plugin where content can be rendered via hooks.
```

Comment:
Would we like to mention hooks (React hooks) here (which is more like the "how") or would we only like to refer to what it makes possible? 

```suggestion
- **Extension point**: A place in Grafana Core or in a plugin where content can be hooked into.
```